src/management/OrganizationsManager.js

```diff
@@ -264,7 +264,8 @@ utils.wrapPropertyMethod(OrganizationsManager, 'delete', 'organizations.delete')
  *   console.log(enabled_connections);
  * });
  *
- * @param   {String}    [organization_id]   Organization ID
+ * @param   {Object}    params        Organization parameters.
```

Comment:
The generated docs is wrong: 
![image](https://user-images.githubusercontent.com/2146903/135079452-a776224d-c2b9-4c90-af3e-838ca0c3992b.png)

As you can see it looks like it expects an optional `organization_id`, while it expects a required `{id: ''}`